scene/gui/rich_text_label.cpp

```diff
@@ -5574,6 +5574,7 @@ void RichTextLabel::set_text(const String &p_bbcode) {
 	}
 	text = p_bbcode;
 	_apply_translation();
+	emit_signal(SNAME("changed_text"), text);
```

Comment:
RTL has a lot of different methods that change the text, `parse_bbcode`, `bbcode`, `add_*`, `push_*` so adding it to `set_text` only will have inconsistent behavior.

Not sure where's the best place to do it, maybe `_add_item()`/`_remove_item` and an additional one in the `add_text` (in a branch where existing text item is updated).

Reply:
I'm unsure of how the entire bbcode system works, but as far as I can tell, the actual value behind `text` of `RichTextLabel` does not get changed anywhere outside of `set_text()`. 

If the value of `text` hasn't changed, the signal `changed_text` should not be emitted.

Reply:
> If the value of text hasn't changed, the signal changed_text should not be emitted.

I guess this makes sense. But it should be at least mentioned in the documentation.

Reply:
> I guess this makes sense. But it should be at least mentioned in the documentation.

I updated the documentation a little so that this is clear